vendor/assets/stylesheets/hint.css

```diff
@@ -0,0 +1,649 @@
+/*! Hint.css - v2.3.2 - 2016-07-28
```

Comment:
I would have liked to have installed this with a gem to avoid checking in this css, but alas, the associated gem is not up to date; i also tried installing via npm and updating rails asset load path to include it that way but alas, their source sass file has a bug that causes a compilation error, hence this file in the vendor directory :)